Beat-Board/README

```diff
@@ -0,0 +1,40 @@
+Authors: Albert Paez | albertzeap@gmail.com
```

Comment:
This readme is not in accordance with the repository. Have a look at this template:
https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md

Reply:
I created a new readme that follows the template